packages/material-ui-codemod/README.md

```diff
@@ -239,6 +307,94 @@ npx @material-ui/codemod@next v5.0.0/collapse-rename-collapsedheight <path>
 
 You can find more details about this breaking change in [the migration guide](https://next.material-ui.com/guides/migration-v4/#collapse).
 
+#### `component-rename-prop`
+
+A generic codemod to rename any component prop.
+
+```diff
+-<Component prop="value" />
+-<Component prop />
++<Component newProp="value" />
++<Component newProp />
+```
+
+<!-- #default-branch-switch -->
+
+```sh
+npx @material-ui/codemod@next v5.0.0/component-rename-prop <path> -- --component=Grid --from=prop --to=newProp
+```
+
+#### `core-styles-import`
+
+Rename private from from `core/styles/*` to `core/styles`
+
+```diff
+-import { darken, lighten } from '@material-ui/core/styles/colorManipulator';
++import { darken, lighten } from '@material-ui/core/styles';
+```
+
+```sh
+npx @material-ui/codemod@next v5.0.0/core-styles-import <path>
+```
+
+#### `create-theme`
+
+Rename the function `createMuiTheme` to `createTheme`
+
```

Comment:
Should we add a diff here and in the other new codemods?

````suggestion

```diff
-import { createMuiTheme } from '@material-ui/core/styles';
+import { createTheme } from '@material-ui/core/styles';
```
````

Reply:
added